pkgs/applications/misc/tango/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "tango";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "masakichi";
+    repo = pname;
```

```suggestion
    repo = "tango";
```

See: https://github.com/nix-community/nixpkgs-lint/issues/21

---

Fixed